docs/platform-services/threat-intelligence-indicators.md

```diff
@@ -248,4 +190,288 @@ Parameters:
 * `hasThreatMatch([srcDevice_ip], confidence > 50 AND source=”FreeTAXII”)`
 * `hasThreatMatch([srcDevice_ip], source=”s1” OR (source=”s2” confidence > 50 AND))`
 * `hasThreatMatch([srcDevice_ip], expired_indicators)`
-* `hasThreatMatch([srcDevice_ip], confidence > 50, all_indicators)`
\ No newline at end of file
+* `hasThreatMatch([srcDevice_ip], confidence > 50, all_indicators)`
+
+## Upload formats
+
+Use the following formats for threat intelligence indicator files when you [add indicators in the Threat Intelligence tab](#add-indicators-in-the-threat-intelligence-tab) or when you use the upload APIs in the [threatIntelIngest](https://api.sumologic.com/docs/#tag/threatIntelIngest) resource:
+
+* [Normalized JSON format](#normalized-json-format)
+* [CSV format](#csv-format)
+* [STIX 2.1 JSON format](#stix-21-json-format)
+
+### Normalized JSON format
+
+Normalized JSON format is a standardized method to present JSON data. You can use this format to load indicators from multiple sources. 
+
+#### Example file
+
+Following is an example threat indicator file in normalized JSON format (from the [uploadNormalizedIndicators API](https://api.sumologic.com/docs/#operation/uploadNormalizedIndicators)):
+
+```
+{
+  "indicators": [
+    {
+      "id": "indicator--d81f86b9-975b-4c0b-875e-810c5ad45a4f",
+      "indicator": "192.0.2.0",
+      "type": "ipv4-addr",
+      "source": "FreeTAXII",
+      "updated": "2023-03-21T12:00:00.000Z",
+      "validFrom": "2023-03-21T12:00:00.000Z",
+      "validUntil": "2023-03-21T12:00:00.000Z",
+      "confidence": 1,
+      "threatType": "indicator",
+      "fields": {
+        "property1": "string",
+        "property2": "string"
+      }
+    }
+  ]
+}
+```
+#### Required attributes
+
+For information about the attributes to use, see ["Indicator" in the STIX 2.1 specification](https://docs.oasis-open.org/cti/stix/v2.1/os/stix-v2.1-os.html#_muftrcpnf89v), and the [uploadNormalizedIndicators API](https://api.sumologic.com/docs/#operation/uploadNormalizedIndicators) in the [threatIntelIngest](https://api.sumologic.com/docs/#tag/threatIntelIngest) resource. 
+
+The following attributes are required:
```

Comment:
Somehow Github's markdown rendering put all these bullets on the same line. But I'm not sure if users read the documentation on Github or if this is getting rendered somewhere else.